plugins/Flanger/QuadratureLfo.h

```diff
@@ -30,51 +30,66 @@
 class QuadratureLfo
 {
 public:
-	QuadratureLfo( int sampleRate );
+	QuadratureLfo( int sampleRate )
+	{
+		setSampleRate(sampleRate);
+	}
 	~QuadratureLfo()
 	{
 	}
 
 	inline void setFrequency( double frequency )
 	{
-		if( frequency < 0 || frequency > ( m_samplerate / 2.0 ) || frequency == m_frequency )
+		if( frequency < 0 || frequency > m_srOverTwo || frequency == m_frequency )
 		{
 			return;
 		}
 		m_frequency = frequency;
 		m_increment = m_frequency * m_twoPiOverSr;
 
-		if( m_phase >= F_2PI )
+		if( m_phase >= D_2PI )
 		{
-			m_phase -= F_2PI;
+			m_phase -= D_2PI;
 		}
 	}
 
 
-
-
 	inline void restart()
 	{
 		m_phase = 0;
 	}
 
 
-
-
 	inline void setSampleRate ( int samplerate )
 	{
 		m_samplerate = samplerate;
 		m_twoPiOverSr = F_2PI / samplerate;
+		m_srOverTwo = samplerate / 2.0;
 		m_increment = m_frequency * m_twoPiOverSr;
 	}
 
-	void tick( float *s, float *c );
+
+	inline void setOffset( double offsetVal )
+	{
+		m_offset = offsetVal;
+	}
+
+
+	void tick( float *s, float *c )
```

Comment:
I think the original intention for those names was to represent sine and cosine functions. It might be better to rename them(maybe `l` and `r`?).